src/wasm-lib/kcl/src/executor.rs

```diff
@@ -47,6 +50,14 @@ pub struct ExecState {
     /// The current value of the pipe operator returned from the previous
     /// expression.  If we're not currently in a pipeline, this will be None.
     pub pipe_value: Option<KclValue>,
+    /// Identifiers that have been exported from the current module.
+    pub module_exports: HashSet<String>,
+    /// The stack of use statements for detecting circular module imports.  If
+    /// this is empty, we're not currently executing a use statement.
+    pub import_stack: Vec<std::path::PathBuf>,
+    /// The directory of the current project.  This is used for resolving import
+    /// paths.
```

Comment:
Could you document when it can be None and what that means? Depending on the answer, it might be better to use a custom enum rather than Option

Reply:
I assume 'none' means current working directory but yeah should be documented

Reply:
I added an enum, so hopefully it's clearer. 282225c1aeb651df73c02c3faa0d5ce11d89bc5f

Reply:
Wait, the `ProjectDirectory` enum that I made is giving me trouble with wasm. `wasm_bindgen` doesn't support them https://github.com/rustwasm/wasm-bindgen/issues/2407. How are we able to use enums with values elsewhere, like here? 

https://github.com/KittyCAD/modeling-app/blob/fbac9935fec173b953280ed640b23ab67a7c76a8/src/wasm-lib/kcl/src/ast/types.rs#L453-L461

 I may switch back to the `Option<String>`.